### DISCUSSION TITLE
r5s是不是跑不满千兆啊…

### DISCUSSION THREAD

**Discussion by jh0072110** [immortalwrt/immortalwrt#1099] [Q&A]
编译的固件感觉都只能跑六百多……

**Comment by skefy** [comment]
最近我也在愁这个事。 光猫改成桥接，OP拨号直接速度减半

**Comment by microka** [comment]
@jh0072110 @skefy 你们看下 https://github.com/openwrt/openwrt/issues/10224 是不是这个问题？我先是用 OpenWrt 23.05 官方固件拨号发现部分测速站点下行测速很慢，之后切换到 ImmortalWrt 21.02.7，下行测速可以跑满2000M带宽，之后再切换到 ImmortalWrt 23.05.0-rc4，下行测速就和 OpenWrt 23.05 一样很慢了。（所有测速均为光猫桥接、wrt拨号）

**Reply by jh0072110** [reply]
这估计得让天灵大佬瞅瞅了，我用隔壁lede编译暂时解决了这个问题…

**Comment by jh0072110** [comment]
https://github.com/immortalwrt/immortalwrt/issues/933#issuecomment-1850780572
这回答是不是对这个讨论也有效（傻笑

**Comment by jh0072110** [comment]
好消息！能跑满了！

**Reply by skefy** [reply]
看路由器运行两天后，网速是否可以稳定
